sources/router.c

```diff
@@ -306,7 +306,9 @@ od_router_status_t od_router_route(od_router_t *router, od_client_t *client)
 {
 	kiwi_be_startup_t *startup = &client->startup;
 	od_instance_t *instance = router->global->instance;
-
+#ifdef LDAP_FOUND
+	od_ldap_server_t *ldap_server = od_ldap_server_allocate();
```

Comment:
When I try to store a pointer to ldap_server in client->ldap_server or rule->ldap_server, I get an error:
/odyssey/sources/client.h:73:9: error: unknown type name ‘od_ldap_server_t’
   73 |         od_ldap_server_t *ldap_server;
How fix it?